packages/hop-node/src/cli/exitCommitTx.ts

```diff
@@ -38,6 +38,9 @@ async function main (source: any) {
   }
 
   const chainSpecificWatcher: ExitWatcher = watcher.watchers[chain]
-  await chainSpecificWatcher.relayXDomainMessage(commitTxHash)
+  const txHashes = commitTxHashes.split(',')
```

Comment:
with above suggestion, `commitTxHashes` would be an array

Reply:
Resolved in 62d3ebb08a4d703cac759d6ab80f7d9dfdabfeac